projects/fsastorefrontlib/src/cms-components/agent/agent-search-list/agent-search-list.component.ts

```diff
@@ -26,16 +29,21 @@ export class AgentSearchListComponent implements OnInit, OnDestroy {
     this.subscription.add(
       this.route.queryParams.subscribe(params => this.initialize(params))
     );
-
     this.searchResults$ = this.agentSearchService.getResults().pipe(
       tap(agents => {
         if (agents) {
+          this.initialActiveAgent = agents.agents[0];
```

Comment:
agents.agents[0] looks a little bit confusing.
You can change the output of this.agentSearchservice.getResults() from agents to result (searchResult.agents[0])